python/packages/ai/teams/app.py

```diff
@@ -3,12 +3,12 @@
 Licensed under the MIT License.
 """
 
-from typing import Awaitable, Callable, Dict, Generic, List, TypeVar
+from typing import Awaitable, Callable, Dict, Generic, List, Optional, TypeVar
 
-from botbuilder.core import Bot, TurnContext
-from botbuilder.core.activity_handler import ActivityTypes
+from botbuilder.core import Bot, BotFrameworkAdapter, TurnContext
```

Comment:
Does the python SDK not have `CloudAdapter`? It's the modern version of `BotFrameworkAdapter`

Reply:
ill look and see, all the python examples used this class so I didn't check if the cloud adapter was available. If it exists ill use that instead.